util_test.go

```diff
@@ -27,3 +28,80 @@ func TestFileExistsReturnsFalseForNonExistingFiles(t *testing.T) {
 	assertEqual(t, fileExists(("util_nope.go")), false)
 	assertEqual(t, fileExists(("util_nope_test.go")), false)
 }
+
+func TestDefaultReleaseSentryReleaseEnvvar(t *testing.T) {
+	releaseVersion := "1.2.3"
+	cleanup := envSetter(map[string]string{
+		"SENTRY_RELEASE": releaseVersion,
+	})
```

Comment:
Why don't you use https://pkg.go.dev/testing#T.Setenv ?

Reply:
I liked the ability to set multiple environment variables in a single call but it's not strictly necessary - I can shift to using  `T.Setenv` instead :)

Reply:
It's a non-issue for me. One thing that would probably be an issue is flakiness on CI as doing set/un-set environment variable within a test scope won't enable the test suite to run in parallel. See the notes on https://pkg.go.dev/testing#T.Setenv

Reply:
in the end both implementations are similar enough, I just moved this to `T.Setenv`